### DISCUSSION TITLE
Editing automations from the UI

### DISCUSSION THREAD

**Discussion by JorgeAnzola** [frenck/home-assistant-config#76] [Q&A] [answered]
Hello, mate!

I have a similar setup to yours, I just accepted the fact that I can't edit automation as they are not in the automations.yaml but I never dug it about it. You also can't edit them? Or there's a workaround? 
Thanks!

**Comment by frenck** [comment, marked as answer]
You can't edit automations from the UI if they are loaded from the split config, however, you can mix and match them.

In my setup, I load both automation created from the UI and my manual created ones. The ones managed by the UI, still work and can be edited via the UI, while one could keep the more complex ones in pure YAML.

**Reply by halogen28372** [reply]
Do you create them and them cut and past the yaml into the proper folder?  I saw some of your automations have "id:" doesn't this mean it was created in the ui?

**Reply by frenck** [reply]
> Do you create them and them cut and past the yaml into the proper folder?

No, I write them manually.

>  saw some of your automations have "id:" doesn't this mean it was created in the UI?

No, it doesn't.
